hugegraph-rocksdb/src/main/java/com/baidu/hugegraph/backend/store/rocksdb/RocksDBStore.java

```diff
@@ -265,7 +260,7 @@ protected RocksDBSessions open(HugeConfig config, String dataPath,
             sessions = this.openSessionPool(config, dataPath,
                                             walPath, tableNames);
         } catch (RocksDBException e) {
-            RocksDBSessions origin = dbs.get(dataPath);
+            RocksDBSessions origin = this.dbs.get(dataPath);
             if (origin != null) {
                 if (e.getMessage().contains("No locks available")) {
                     // Open twice, but we should support keyspace
```

Comment:
update comments:
Open twice, copy a RocksDBSessions reference, since from v0.11.2 release we don't support multi graphs share rocksdb instance (before v0.11.2 graphs with different CF-prefix share one rocksdb instance and data path), so each graph has its independent data paths, but multi CFs may share same optimized disk(or optimized disk path).